examples/common/js/jsonlint.js

```diff
@@ -0,0 +1,418 @@
+/* Jison generated parser */
```

Comment:
I did actually.  As it exists on npm, jsonlint [won't work](https://github.com/zaach/jsonlint/issues/57) with webpack.  The library is small, so I copied it here and removed the offending code.